.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Playwright",
+    "image": "mcr.microsoft.com/playwright:next",
+    "postCreateCommand": "npm install && npm run build",
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
```

Comment:
Curious to know why this is required?

Reply:
By default it uses `/bin/sh` which is not very user friendly. On the SuperLinter project of GitHub they are doing that too: https://github.com/github/super-linter/blob/master/.devcontainer/devcontainer.json